app/assets/stylesheets/views/_covid.scss

```diff
@@ -10,43 +11,73 @@ $covid-green: #7eff8e;
 }
 
 .covid__page-header {
-  padding-bottom: govuk-spacing(6);
   margin-bottom: govuk-spacing(8);
   border-top: solid govuk-spacing(2) $covid-green;
   background-color: govuk-colour("black");
   color: govuk-colour("white");
 }
 
-.covid__logos {
-  max-width: 100%;
+.covid__page-header-light {
+  padding-top: govuk-spacing(3);
+  padding-bottom: govuk-spacing(1);
+  background: $covid-grey;
+
+  @include govuk-media-query($from: tablet) {
+    padding-bottom: govuk-spacing(3);
+  }
+}
+
+.covid__logo-wrapper {
   margin-bottom: govuk-spacing(6);
 
   @include govuk-media-query($from: tablet) {
-    max-width: 75%;
+    margin-bottom: govuk-spacing(8);
   }
+}
 
-  @include govuk-media-query($from: desktop) {
-    max-width: 50%;
+.covid__logos {
+  max-width: 100%;
+  height: auto;
+
+  @include govuk-media-query($from: tablet) {
+    margin-top: govuk-spacing(2);
   }
 }
 
 .covid__page-header-link {
+  font-weight: normal;
+
   &:link,
   &:visited {
     color: govuk-colour("white");
   }
   &:focus {
     color: $govuk-text-colour;
   }
+}
+
+.covid__highlight-link {
```

Comment:
The green SVG on yellow focus is a bit painful - perhaps the focus state should only be on the text? 

<img width="501" alt="Screenshot 2020-03-27 at 18 28 00" src="https://user-images.githubusercontent.com/29889908/77788220-c56e6200-7058-11ea-9290-7c0434568cf3.png">

Reply:
Ah yes, I'd forgotten about that. Okay, good point, will take a look.

Reply:
@vanitabarrett have fixed.

Reply:
Looks awesome! 👏